packages/shared-components/src/template/templates/navdesign/navCheckbox/form.ejs

```diff
@@ -6,9 +6,9 @@
     {% for (var attr in ctx.input.attr) { %}
     {{attr}}="{{ctx.input.attr[attr]}}"
     {% } %}
-    {% if (ctx.checked) { %}checked=true{% } %}>
-        {{ctx.input.content}}
```

Comment:
Regner med det var meningen å fjerne bruken av `ctx.input.content`? Vi bruker ikke det for navCheckbox?

Reply:
Ja, den virker ikke relevant for oss. Og native checkboxes i Formio bruker ikke denne templaten.